components/sync_manager/src/syncmanager.udl

```diff
@@ -54,6 +54,7 @@ enum SyncReason {
     "PreSleep",
     "Startup",
     "EnabledChange",
+    "Backgrounded",
```

Comment:
Could this be a breaking change in android? in the case where android depends on an exhaustive `when` expression that checks all the variants(I'm not sure if does or doesn't, just making sure we mark it as such if it is 😬)

Reply:
When I built android components locally against these changes while creating the `SyncManagerDeviceType` change, no `SyncReason` errors arose. It's worth noting that android defines it's own sync reasons and converts them to our sync reasons as needed (see [here](https://github.com/mozilla-mobile/firefox-android/blob/9de5ab8cc098674aad5190ce8b00ae6be65e9bd0/android-components/components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/sync/Types.kt#L44).

Reply:
Also please feel free to build a-c locally and check my math!

Reply:
No makes sense! Thank you for checking 🙏